docs/install/scaffold.md

```diff
@@ -35,28 +34,23 @@ invenio-cli init rdm -c <version>
 
 Where ``<version>`` should be replaced with the version you want to install, e.g.:
 
-- LTS release (for production systems): ``v9.1``
-- STS release (for feature previews): ``v10.0``
+- Current release (for production systems): ``v12.0``
 
 ``` console
 Initializing RDM application...
 Running cookiecutter...
 project_name [My Site]:
 project_shortname [my-site]:
 project_site [my-site.com]:
```

Comment:
Here for v12 (and probably before) I have:

```suggestion
project_name [My Site]:
project_shortname [my-site]:
package_name [my_site]:
project_site [my-site.com]:
```

`package_name` is new. Let's add it as well.